drama-queen/src/ui/components/orchestrator/Orchestrator.tsx

```diff
@@ -52,12 +52,12 @@ export function Orchestrator(props: OrchestratorProps) {
   const { t: t2 } = useTranslation('modalMessage')
   const { onChange, ref } = useAutoNext()
 
-  // get the initial data for useLunatic
-  const initialData = surveyUnit?.data as LunaticData | undefined
-
   // the given surveyUnit can be empty or partial, we initialize it for having the waited format
   const initialSurveyUnit = getinitialSurveyUnit(surveyUnit)
 
+  // get the initial data for useLunatic
+  const initialData = getInitialData(initialSurveyUnit) as LunaticData
```

Comment:
I am not a fan of using the `as` keyword in this use case, because getInitialData can return LunaticData or undefined. 
I understand that in this case, you are pretty sure the value can only be of type LunaticData... but can maybe a better way. 

Maybe the `getInitialData` should return `LunaticData` ? And we must manage the case where the `surveyUnit.data` is undefined ?

Reply:
You're right, I'll fix this with your suggestions before accepting the dev.